src/python/WMCore/WMRuntime/NATSInterface.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+"""
+NATSInfo provides wrapper around NATS to send WMAgent messages
+"""
+from __future__ import division
+from __future__ import print_function
+
+# system modules
+import socket
+
+# WMCore modules
+from WMCore.WMSpec.WMStep import WMStepHelper
+from WMCore.WMSpec.WMWorkload import getWorkloadFromTask, WMWorkloadHelper
+from WMCore.Storage.SiteLocalConfig import loadSiteLocalConfig, SiteConfigError
+from WMCore.WMRuntime.Bootstrap import getSyncCE
+
+# CMSMonitoring modules
+from CMSMonitoring.NATS import NATSManager
```

Comment:
Importing from CMSMonitoring from the worker node(?) I'm actually surprised that it's in the job environment...

Reply:
NATS codebase belongs to CMSMonitoring repository, that's why is the import.